arm-containerregistry/2017-06-01-preview/swagger/containerregistry.json

```diff
@@ -121,8 +121,8 @@
         "tags": [
           "Registries"
         ],
-        "description": "Creates a container registry with the specified parameters.",
-        "operationId": "Registries_Create",
+        "description": "Creates or updates a container registry with the specified parameters.",
+        "operationId": "Registries_CreateOrUpdate",
```

Comment:
I'm not sure why you need ```CreateOrUpdate``` and ```Update``` operations. It would make sense to have ```Create``` and ```Update``` as separate operations if the Update does more than just update tags. Else, you can simply remove the update operation altogether since CreateOrUpdate already does the updating. This applies to other operations below too.